addons/hr_holidays/static/src/core/common/composer_patch.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.Composer.suggestionPartner" t-inherit-mode="extension">
+        <xpath expr="//span[@t-if='partner.email']" position="before">
+            <span class="text-warning me-1 small" t-if="partner.OutOfOfficeDateEndText">
+                <i class="fa fa-plane" role="img"/> <t t-esc="partner.OutOfOfficeDateEndText"/>
+            </span>
+        </xpath>
+    </t>
```

Comment:
<img width="641" alt="Screenshot 2025-03-27 at 11 25 46" src="https://github.com/user-attachments/assets/c10460c7-518b-4c2c-a299-822be94beaaf" />

Plane icon is shown twice, "back in" label is too light.
I would not show the plane next to "back in" label, and I would put `.fw-bold` so this is more visible.
This would look like this:

<img width="640" alt="Screenshot 2025-03-27 at 11 27 22" src="https://github.com/user-attachments/assets/1bd3656b-12da-4863-8f40-6e0cf95e1a58" />